src/utils/logger.ts

```diff
@@ -0,0 +1,10 @@
+import { createLogger } from "@companieshouse/structured-logging-node";
```

Comment:
there is already src/main/utils folder. Can all the file from src/utils be move there please?